server/users.js

```diff
@@ -987,6 +986,14 @@ class User extends Chat.MessageContext {
 		if (isForceRenamed) this.trackRename = oldname;
 		return true;
 	}
+	getUpdateuserText() {
+		const named = this.named ? 1 : 0;
+		const settings = {blockPMs: this.ignorePMs, blockChallenges: this.blockChallenges};
```

Comment:
We might want to rename `ignorePMs` to `blockPMs` while we're making this change.

Reply:
I also wonder if we should start bundling user settings together, instead of having them on the top level of `User`. We have a bunch more (although most of them probably shouldn't be sent here).

Reply:
> We might want to rename `ignorePMs` to `blockPMs` while we're making this change.

Done

> I also wonder if we should start bundling user settings together, instead of having them on the top level of `User`. We have a bunch more (although most of them probably shouldn't be sent here).

I think that's maybe out of scope and can be done in a followup by someone?

Reply:
Oh yeah the latter one was definitely not a suggestion for this PR.